sdk/identity/azure-identity/test/live/client_secret_credential.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <azure/identity/client_secret_credential.hpp>
+
+#include <gtest/gtest.h>
+
+#include <chrono>
+#include <cstdlib>
+
+inline std::string GetEnv(std::string const& name)
+{
+#if defined(_MSC_VER)
+#pragma warning(push)
+// warning C4996: 'getenv': This function or variable may be unsafe. Consider using _dupenv_s
+// instead.
+#pragma warning(disable : 4996)
+#endif
+  auto const result = std::getenv(name.c_str());
+  return result != nullptr ? result : "";
+#if defined(_MSC_VER)
+#pragma warning(pop)
+#endif
+}
+
+TEST(ClientSecretCredential, Basic)
```

Comment:
nit. Consider giving a better name to the test.  `Basic` is not telling much.  Probably something like `GetToken` or `GetBasicToken`

Reply:
"Basic" means the "the most common and straightforward way of using the credential" - i.e. with just providing the required parameters and not tweaking anything else.